igcollect/jmx.py

```diff
@@ -59,6 +60,8 @@ def main():
                 port,
                 '--thread-prefixes',
                 thread_prefixes,
+                '--metric-file',
+                args.metric_file,
```

Comment:
This will also add the argument to all current users. Is that fine (is it the implicit default) ?